community/bolt/src/main/java/org/neo4j/bolt/v1/messaging/PackStreamMessageFormatV1.java

```diff
@@ -108,7 +108,8 @@ public void handleRunMessage( String statement, Map<String,Object> params )
         {
             packer.packStructHeader( 2, MessageTypes.MSG_RUN );
             packer.pack( statement );
-            packer.packRawMap( params );
+            packer.packParameterMap( params );
```

Comment:
I was thinking that it might be more valuable to check if we receive node/rel/path as input parameters, which means we should not accept node/rel/path in reader?